lib/OptionMenu/MenuItem.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { string, func, bool } from 'prop-types';
+import cx from 'classnames';
+import Icon from '../Icon';
+
+function MenuItem({ text, onClick, iconName, selected }) {
+  const classes = cx(
+    'my-1 rounded-small w-48 border-0 flex items-center py-1 px-2 hover:bg-neutral-95 focus:outline-none focus:shadow-blue-focus-sm',
+    {
+      'text-neutral-20': !selected,
+      'text-blue-primary': selected
+    }
+  );
+
+  return (
+    <button className={classes} type="button" onClick={onClick}>
```

Comment:
💚 i think we should chuck in a `...rest` here too. I think its always handy for things like buttons. Maybe disabled could come in handy too?